usagov/vote-gov-drupal
# Qa/vote 977 email signup

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-977](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-977&assignee=611272e0eb7aef00694b8a97)

## Description (optional)
This PR will add test coverage for the email signup component on the home page 

## Deployment and testing (required, if applicable)
### Testing steps
1. pull down branch 
2. start local 
3. cd into `testing`
4. run `npm run cy:createUser`
5. run `npm run cy:open` 
6. run `email-sign-up.cy.js` and verify that the test is passing 
7. close out of cypress and in terminal run `npm run cy:deleteUser`
